src/Game.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+class Game extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      score: 0,
+      current: 1,
+      next: 2,
+    };
+  }
+
+  handleClick = () => {
+    this.setState({
+      score: this.state.score + this.state.current,
+    });
+  };
+
+  addPoint = () => {
+    if (this.state.score < 10) {
+      alert(`You can't afford that!`);
+    } else {
+      this.setState({
+        score: this.state.score - 10,
+        current: this.state.current + 1,
+        next: this.state.next + 1,
+      });
+    }
+  };
+  reset = () => {
```

Comment:
Remember to keep it simple. You could reset the state as seem below:

`this.setState({
      score: 0,
      current: 1,
      next: 2,
    })`